x-pack/plugin/rollup/src/test/java/org/elasticsearch/xpack/rollup/job/RollupIndexerStateTests.java

```diff
@@ -572,6 +571,7 @@ public void testMultipleJobTriggering() throws Exception {
             latch.countDown();
             assertBusy(() -> assertThat(indexer.getState(), equalTo(IndexerState.STOPPED)));
             assertTrue(indexer.abort());
+            assertTrue(indexer.getStats().getNumInvocations() >= 6);
```

Comment:
nit: maybe use Matchers.greaterThanOrEqualTo?

Reply:
I was actually looking for something like that and missed it somehow.